dotCMS/src/main/java/com/dotmarketing/portlets/contentlet/business/HostAPI.java

```diff
@@ -20,6 +20,8 @@
  *
  */
 public interface HostAPI {

```

Comment:
![CRITICAL](https://sonarsource.github.io/sonar-github/severity-critical.png 'Severity: CRITICAL') Move constants to a class or enum. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1214)